src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -56,7 +55,57 @@ public class DefaultHtmlGenerator : IHtmlGenerator
             IModelMetadataProvider metadataProvider,
```

Comment:
My question was about `[Obsolete]` versus leaving this constructor's `public` surface as-is (other than fixing the incorrect doc comment).